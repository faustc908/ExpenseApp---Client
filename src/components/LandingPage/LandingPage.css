.Appdesc {
    float: center;
    margin-left: 100px;
    margin-right: 100px;
    margin-top: 75px;
    max-width: 70%;
    font-style: italic;
    font-size: 20px;
    padding-bottom: 220px;
}


@media (max-width: 320px) {
    .Appdesc{
      font-size: 17px;
      min-width: 80%;
      margin: 10px;
      margin-left: 25px;
      float:center;
    }
  }