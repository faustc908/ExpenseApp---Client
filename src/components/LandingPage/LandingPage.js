import React from 'react';
import './LandingPage.css'


//  Landing page with application description

const LandingPage = () => {
    return (
        <div className='Appdesc'>
            <h2 className='desc'>
                Welcome to ExpenseApp
            </h2>
            <p className='desc'>
                Current times are tough and keeping track of your expenses is as important as ever.  ExpenseApp allows you to calculate your daily transactions so you can keep track of and see where your money is spent.  It also allows you to record any relevant information on a database so you can keep track of disposable income as well as any notes you want to add on good deals or nuances in your daily or monthly expenses.  This app was designed to give you more control in uncertain times.
            </p>
        </div>
    )
}

export default LandingPage;