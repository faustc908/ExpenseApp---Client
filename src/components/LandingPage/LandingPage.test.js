import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingPage from './LandingPage'
import ReactDOM from 'react-dom';

describe(`LandingPage component`, () => {

    it('renders a LandingPage by default', () => {
        const wrapper = shallow(<LandingPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

})