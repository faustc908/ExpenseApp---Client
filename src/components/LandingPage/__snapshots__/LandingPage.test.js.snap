// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders a LandingPage by default 1`] = `
<div
  className="Appdesc"
>
  <h2>
    Welcome to ExpenseApp
  </h2>
  <p>
    Current times are tough and keeping track of your expenses is as important as ever.  ExpenseApp allows you to calculate your daily transactions so you can keep track of and see where your money is spent.  It also allows you to record any relevant information on a database so you can keep track of disposable income as well as any notes you want to add on good deals or nuances in your daily or monthly expenses.  This app was designed to give you more control in uncertain times.
  </p>
</div>
`;
